scheduler/reconcile_test.go

```diff
@@ -1320,12 +1320,13 @@ func TestReconciler_RescheduleLater_Batch(t *testing.T) {
 		place:             0,
 		inplace:           0,
 		attributeUpdates:  1,
-		stop:              0,
+		stop:              1,
```

Comment:
It's worth noting that in most tests `stop` field should match `place+attributeUpdates` like in most of reconcile_tests.go files.

In general, if only rescheduling is at play, `place` should capture the `rescheduledNow` allocs while `attributeUpdates` seems to capture `rescheduledLater` allocs in the reconciler code.